test/sql/p3.20-window_function.slt

```diff
@@ -0,0 +1,88 @@
+# 4 pts
+# The real test process begins...
+
+# Create a table
+statement ok
+create table t1(v1 int);
+
+# Simple window aggregation over the table
+query
+insert into t1 values (-99999), (99999), (0), (1), (2), (3);
+----
+6
+
+query
+select count(*) over (), min(v1) over (), max(v1) over (), count(v1) over (), sum(v1) over () from t1;
+----
```

Comment:
do we require order on the output? if not, we can use `query nosort` to skip checking the data order.